app/assets/stylesheets/content/_forms.sass

```diff
@@ -232,12 +232,17 @@ fieldset.form--fieldset
       @include grid-size(expand)
       // override max-width set by including grid-content
       max-width: 100%
-      padding-left: 0.5rem
+      padding: 0.5rem
```

Comment:
Well, can we change it back because I find it weird that we would need a top/border padding in this case.